guidelines/content-guidelines/01-content-strategy-k2.md

```diff
@@ -0,0 +1,193 @@
+
+### Task (tutorial) topics
+
+Provide "how-to" instructions that enable users to accomplish a task. Each task topic should tell how to perform a single, specific procedure.
+
+Select a title that describes the task that's accomplished, not the documented software feature - for example, "Define ressource consumption", not "Select a profile". You can use gerund form "Selecting...", imperative "Select...", or "How to select...".
+
+With regards to structure, it’s nice to have an **introductory paragraph** ("why would I want to do this task?"), **prerequisites** if needed, then the **steps**, and finally the expected **result** that shows the operation was successful.
+It's good practice to have 5-9 steps; anything longer can probably be split.
+
+*For longer code blocks, I'd like to use an expandable section as described [here](https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab) - but not sure whether our website supports this.*
+
+> [- Dita Task Topics](http://docs.oasis-open.org/dita/dita/v1.3/errata02/os/complete/part3-all-inclusive/archSpec/technicalContent/dita-task-topic.html)
+> A task information type answers the "How do I?" question by providing precise step-by-step instructions detailing the requirements that must be fulfilled, the actions that must be performed, and the order in which the actions must be performed. The task topic includes sections for describing the context, prerequisites, expected results, and other aspects of a task.
+
+You can use the [task topic template](../templates/resources/task.md).
+
+### Reference topics
+
+Typically organized into one or more sections containing a list or table with data that is typically “looked up” rather than memorized.
+
+Use nominal style for the title, for example, "Configuration Parameters".
+
+*In our case, architecture diagrams could fall into this category – we could also choose to define them as a separate type – TBD.*
+
+> [- DITA Reference Topics](http://docs.oasis-open.org/dita/dita/v1.3/errata02/os/complete/part3-all-inclusive/archSpec/technicalContent/dita-reference-topic.html)
+> Reference topics provide quick access to fact-based information. In technical information, reference topics are used to list product specifications and parameters, provide essential data, and provide detailed information on subjects such as the commands in a programming language.
+
+### Troubleshooting topics
+
+Provide a condition that the reader may want to correct, followed by one or more descriptions of its cause and suggested remedies.
+
+As title, mention the symptom that needs fixing ("Cannot access...") or the error message. To quote an error message, start and end with `'` to escape `"`. for example, `title: '"FAILED" status for created ServiceInstances'`. Do not use the cause as title ("Incompatible version"), because we also want to help users who have no idea about the cause and only know something's not working as expected.
```

Comment:
I'm wondering if we're not going to shoot ourselves in the knee with providing the error in the title. 
On one hand, if we do this, it's great for the user 'cause they can find it easily when they encounter the problem. 
On the other hand:
- the user has the search tool and can find it easily anyway
- it's problematic and not very elegant when it comes to title formatting/rendering
- one cause can result in multiple errors, one error can have multiple causes, so these docs might grow if you want to describe it all in one doc.

I'd personally stick to using the cause as a title.

Reply:
I agree there is the search tool, but a good search tool shouldn't replace meaningful titles. 

Some users might prefer the search, others will rather scan the titles in the navigation. And as a user, I wouldn't know which cause to look out for (if I knew it, I'd probably be competent to fix it without our doc). So I'd prefer to use the problem for the title.

How about we ask others - the TWs, as well as potential users? What do our devs think, or the SREs? I'm fine with what the majority prefers.